connectors/google/google-sheets/src/main/java/io/camunda/connector/gsheets/util/LetterNumericSystemConverter.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. Licensed under a proprietary license.
+ * See the License.txt file for more information. You may not use this file
+ * except in compliance with the proprietary license.
+ */
+package io.camunda.connector.gsheets.util;
+
+public class LetterNumericSystemConverter {
+
+  private static final int BASE = 26;
+  private static final int SHIFT = 64;
+
+  private LetterNumericSystemConverter() {}
+
+  public static int stringToIn(String number) {
```

Comment:
Let's give even better name, to avoid confusion between `Integer.parseInt(...)`. How about `spreadsheetLetterToNumericIndex`. 
Also, let's add a comment over the method.
Also, please take a look whether the library doesn't have existing functionality for that yet, because I would expect to have it alreay.

Reply:
I found nothing that would implement such functionality